### PR TITLE
Use workspace full and copy things from ops

### PR DESCRIPTION
**Pull request by mads-hartmann**
## Description

The current workspace image didn't have Git which made it really tedious to make changes in Gitpod (you'd have to install git and copy over your config from another working workspace)

This uses workspace full and copies over things from ops based on a bit of trail an error.

I'm really hoping we can archive this repo entirely soon and move the everything to the ops repo so I consider this temporary.

## Related Issue(s)

No issue

## How to test

Open a workspace.

## Release Notes

```release-note
NONE
```

## Documentation

N/A

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mads-hartmann** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by ArthurSens — APPROVED** [review]

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/67#pullrequestreview-871913224" title="Approved">ArthurSens</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)~~ [ArthurSens]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: e31b93541695082b5303293d4f51fe5712a5203f</details>

**Pull request merged**
